src/runtime-rs/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[workspace]
+members = [
+    "crates/shim",
+]
+
+[patch.'crates-io']
+containerd-shim-protos = { git="https://github.com/containerd/rust-extensions.git", branch = "main", features = ["async"]}
```

Comment:
build warning: 
patch for `containerd-shim-protos` uses the features mechanism. default-features and features will not take effect because the patch dependency does not support this mechanism`

Reply:
Thanks for the review, fixed